pylossless/pipeline.py

```diff
@@ -111,7 +127,7 @@ def marks_array2flags(inarray, flag_dim='epoch', init_method='q', init_vals=(),
                       init_dir='both', init_crit=(), flag_method='z_score',
                       flag_vals=(), flag_crit=(), trim=0):
 
-    ''' This function takes an array typically created by chan_variance or
+    ''' This function takes an array with typically created by chan_variance or
```

Comment:
I think the new "with" here does not belong?

Reply:
Yes, that may have been just due to the mess of rebasing and all. I did not intend to add the with. I probably removed it, and a commit due to rebasing put it back. It was a bit of a mess. Anyway, I'll squash all this commit when I merge.